assets/content/content.json

```diff
@@ -8,8 +8,8 @@
         "avatar": "Avatar",
         "testusername": "Rob Johnson",
         "logout": "Logout",
-        "help": "Help", 
-        "helpLink": "https://apps.nrs.gov.bc.ca/int/confluence/display/VHERS/PIN+System+Help+Materials"
+        "help": "Submit a Support Ticket", 
+        "helpLink": ""
```

Comment:
Maybe throw a link here to somewhere (anywhere) just so it doesn't look like it doesn't do anything?

Reply:
Since we have the following onclick function:
`	const onClickHandle = () => {
		window.open(Text.header.helpLink, "_black", "noreferrer")
	}`

It should still open to a blank page (about:blank) in a new tab

Reply:
Ah didn't see that. That should work